### PR TITLE
Add followers and 'vote' button to posts

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity I added the "vote" button to the heart icon and vote count in the meta text at the top of posts. This is pretty rough ux-wise I think, but thought I'd put it there for now. Let me know if you know where you'd like it to be instead.

File: src/components/Post.js [hunk 1]
```diff
@@ -223,3 +230,51 @@ const ExpandedPostDetails = props => {
...
+  if (meInFollowers) {
+    followersNotMe = without(followers, meInFollowers)
+  } else {
+    followersNotMe = followers
+  }
+
+  let firstTwoFollowersNotMe = followersNotMe.slice(0, 2)
+
+  if (followers.length > 0 && !onlyAuthorIsFollowing) {
+    return <div className='meta followers'>
```
Comment (levity) [inline src/components/Post.js:248]: i'm going to take a crack at refactoring this for readability

Comment (robbiecarlton) [inline src/components/Post.js:248]: :) 
I tried, and then decided to do a direct copy from the old site for accuracy. I'm happy to try a refactor, now that I've got it working the old way

**Comment by levity** [comment]
:+1:

**Pull request merged**